src/main/java/mil/dds/anet/threads/AnetEmailWorker.java

```diff
@@ -131,19 +132,19 @@ protected void runInternal(Instant now, JobHistory jobHistory, Map<String, Objec
     emailContext.put("SUPPORT_EMAIL_ADDR", config.getDictionaryEntry("SUPPORT_EMAIL_ADDR"));
     emailContext.put("dateFormatter",
         DateTimeFormatter.ofPattern((String) config.getDictionaryEntry("dateFormats.email.date"))
-            .withZone(DaoUtils.getDefaultZoneId()));
```

Comment:
Through the code, we now use both:
`ZoneId.systemDefault()`
and
`DaoUtils.getDefaultZoneId()`
As a result, it is not clear what the intent is and what is used when.
I propose to have 2 methods in DaoUtils:
`DaoUtils.getServerLocalZoneId()` -> delegates to ZoneId.systemDefault() and is used to format user-facing content such as emails
`DaoUtils.getServerNativeZoneId()` -> same behavior, and instead of, the currentl `DaoUtils.getDefaultZoneId()` and is used to define the native date representation for the database.